name: "jackal"
N: 20
integrator_step: 0.05

# solver: "acados" # acados or forces
solver: "forces" # acados or forces

control_frequency: 20
num_obstacles: 3
robot_radius: 0.325
obstacle_radius: 0.725

contouring:
  num_segments: 5
  contour: 2.
  lag: 0.1

weights:
  goal: 10.
  velocity: 2.0 #0.3
  acceleration: 0.05
  angular_velocity: 0.05 #0.05

reference_velocity: 2.
